lib/trento/databases/lifespan.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Trento.Databases.Lifespan do
```

Comment:
This is currently just using the DefaultLifespan. Is it going to be changed with regards to rollup event for databases?

Reply:
Yeah, rollup will come in a separate pr later on